hera_cal/xtalk_filter.py

```diff
@@ -128,21 +128,76 @@ def load_xtalk_filter_and_write(infilename, calfile=None, Nbls_per_load=None, sp
                                clobber=clobber, add_to_history=add_to_history,
                                extra_attrs={'Nfreqs': len(freqs), 'freq_array': np.asarray([freqs])})
     else:
-        for i in range(0, len(hd.bls), Nbls_per_load):
+        nwrites = 0
```

Comment:
Yeah, I originally added this when I was trying to use partial i/o and thought I would be writing the baseline / ant1 / ant2 arrays every partial write but since that didn't pan out, all that i'm trying to save on are frequencies which is kind of unecessary. I'll remove this.